source/data-model-partials/data-model.md

```diff
@@ -0,0 +1,99 @@
+
+These charts and tables are stored in the `dimension_chart` and `dimension_table` tables (named as such because `table`
+is a reserved word in SQL). These tables have a similar structure to one another. They each store information about the
+ethnicity `classification` used by the chart or table, as well as whether the data includes aggregate figures for all, 
+parent ethnicities and unknown ethnicity. They also have a `settings_and_source_data` JSON field, which stores the
+form data from the chartbuilder or tablebuilder used to create the object, and then the `chart_object` or `table_object`
+itself, which is the thing displayed on the measure page (for charts this is a Highcharts object; for tables it is our
+JSON format that our page templates know how to render).
+
+**Classifications**
+Entries in the `classification` table represent the various different sets of ethnicites used by the datasets we collect
+and publish. We currently store classifications for each new chart and table that is saved, and these chart and table
+classifications are used to assign a classification to the dimension itself. These dimension-level classifcations are
+stored in the `dimension_categorisation` table (this table is not shown in the diagram above).  This is a "legacy" table
+that should at some point be removed: dimension-level classifications used to be manually chosen by users as part of 
+authoring a dimension, but now we have the mechanism in place to derive these from the chart and table on the dimension 
+the `dimension_categorisation` table is superfluous.
+
+**Data sources**
+A measure version can have multiple data sources, but our application code currently only allows two (a primary source
+and secondary source).  A `data_source` is published by an `organisation`, is of a `type_of_statistic` (national, 
+official, non-official or experimental) and has a `frequency_of_release`. 
+
+There are also various other pages in the site that do not sit within this topic hierarchy: 
+
+* The [list of ethnic groups](https://www.ethnicity-facts-figures.service.gov.uk/ethnic-groups)
+* The [privacy policy](https://www.ethnicity-facts-figures.service.gov.uk/privacy-policy)
+* Various [dashboard pages](https://www.ethnicity-facts-figures.service.gov.uk/dashboards)
+
+These are not modelled in the database, they are stand-alone pages in the Publisher app.
+
+## Dashboard pages and materialized views
+The data for many of the dashboard pages comes from a set of [materialized views](https://www.postgresql.org/docs/9.5/rules-materializedviews.html)
+that are refreshed every time a new page is published, and every night. Any data migrations that alter the structure of
+the `measure_version` table will need to drop the materialized views before running and re-create them at the end.
+
+## Users and access control
+We have four types of `users`: DEV, ADMIN, RDU and DEPT.  Each `user_type` is associated with a set of default 
```

Comment:
Maybe briefly call out (more obviously) that `capabilities` can be added/removed from the default set on a per-user basis?